middleware/cache/cache_test.go

```
@@ -918,6 +918,89 @@
 	}
 }
 
+func Test_Cache_UncacheableStatusCodes(t *testing.T) {
+	t.Parallel()
+	app := fiber.New()
+	app.Use(New())
+
+	app.Get("/:statusCode", func(c fiber.Ctx) error {
+		statusCode, err := strconv.Atoi(c.Params("statusCode"))
+		if err != nil {
+			return err
```

golangci-lint (1.62.2)

error returned from external package is unwrapped: sig: func strconv.Atoi(s string) (int, error)

(wrapcheck)